src/session/session_dhandle.c

```diff
@@ -269,7 +269,7 @@ __wt_session_release_btree(WT_SESSION_IMPL *session)
 			__session_discard_dhandle(session, dhandle_cache);
 	}
 
-	if (F_ISSET(btree, WT_BTREE_BULK)) {
+	if (btree != NULL && F_ISSET(btree, WT_BTREE_BULK)) {
```

Comment:
Layering wise I think it'd be nice if the btree specific code here were pulled into a separate function here.